### PR TITLE
Add Blaze to FAQ and Use case pages

### PR DESCRIPTION
**Pull request by jhaaaa**
Preview links:

- [FAQ](https://junk-range-possible-git-add-blaze-xmtp-labs.vercel.app/docs/faq#is-there-a-way-to-get-a-list-of-all-current-xmtp-enabled-wallets)

- [Automation use case](https://junk-range-possible-git-add-blaze-xmtp-labs.vercel.app/docs/use-cases/automation)

- [Support use case](https://junk-range-possible-git-add-blaze-xmtp-labs.vercel.app/docs/use-cases/support)

- [Marketing use case](https://junk-range-possible-git-add-blaze-xmtp-labs.vercel.app/docs/use-cases/marketing)

### CODE REVIEW

File: docs/use-cases/support.mdx [hunk 1]
```diff
@@ -12,6 +13,10 @@ Customer support is an integral part of any organization, serving as the bridge
 XMTP‍ presents a new opportunity to revolutionize customer support by providing encrypted communication channels. By leveraging the features of XMTP, customer support can be taken to new heights, balancing both user convenience and information security.
 
 <div class="chain-list white">
+  <a href="https://www.withblaze.app/" class="chain-item">
```
Comment (daria-github) [inline docs/use-cases/support.mdx:16]: nit: do you want to add a `target="_blank"` here to open this in a separate tab?

File: docs/use-cases/support.mdx [hunk 2]
```diff
@@ -12,6 +13,10 @@ Customer support is an integral part of any organization, serving as the bridge
...
+    <img src={blaze} />
```
Comment (daria-github) [inline docs/use-cases/support.mdx:17]: an alt tag would be good here, e.g. "Blaze logo"

**Review by daria-github — APPROVED** [review]

File: docs/use-cases/support.mdx (hunk 2 above)
Comment (jhaaaa) [reply docs/use-cases/support.mdx]: LOVE - thank you so much, @daria-github - added for all of the logos. I will also be sure to add a pass on the entire site. <3

File: docs/use-cases/support.mdx (hunk 1 above)
Comment (jhaaaa) [reply docs/use-cases/support.mdx:16]: Love this - thank you and wonder of wonders - Docusaurus opens external links in a new tab by default! <3

**Pull request merged**